Dockerfile

```diff
@@ -26,17 +26,17 @@ RUN wget http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz \
 # Bind mounts on WSL are flaky 
 # so we must copy into a volume.
 COPY . /app
+RUN  sed -i "s|host: 'localhost'|host: 'mongodb'|g" src/conf.sample.ts
```

Comment:
Docker builds containers on top of containers with each `step` in the `Dockerfile` so subsequent steps would have to be rebuilt if this step were modified. It'd make more sense to place it later in the file.

Also, might be a better idea to use `ENV` vars instead.

Reply:
About the environment variable I agree. About the step I would agree to move it between dependency installation and compiling.

Reply:
i have now moved the sed right before compiling the Typescript.
I am currently not sure how to implement it with an enviroment variable, since the needed information is defined in the docker-compose, which comes into place later.
Maybe you could support here @jraviotta ?